cmd/immuadmin/command/init.go

```diff
@@ -18,7 +18,7 @@ import (
 	"strings"
 
 	"github.com/codenotary/immudb/pkg/client"
-	"github.com/codenotary/immudb/pkg/gw"
+	"github.com/codenotary/immugw/pkg/gw"
```

Comment:
some additional refactoring might be needed around immuadmin as well, immugw should not be a dependency neither...

Reply:
Immuadmin need immugw because it manages immugw service installing procedure. So it need to call some immugw api to, for example, install man pages.
A solution could be to move out also immuadmin, but this is out of scope for the moment.